package.json

```diff
@@ -6,10 +6,10 @@
   ],
   "packageManager": "yarn@3.4.1",
   "devDependencies": {
-    "@react-native-community/eslint-config": "3.2.0",
-    "eslint": "8.26.0",
-    "prettier": "2.8.8",
-    "typescript": "4.9.5"
+    "@rnx-kit/eslint-plugin": "^0.5.0",
+    "eslint": "^8.26.0",
+    "prettier": "^2.8.8",
+    "typescript": "^5.0.0"
```

Comment:
can we jump to 5.3?

Reply:
This is resolves to 5.3 in `yarn.lock`. Or did you want to explicitly specify 5.3 here? We can do that as well if you want to.

Reply:
Yes please, I prefer to see "base" version set in package.json, rather than checking it in yarn.lock